### PR TITLE
Add updated readme with versions

### PR DESCRIPTION
**Pull request by m-langer**

### CODE REVIEW

**Review by HafnerMichael — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -57,10 +57,10 @@ Images
 
     ```sh
```
Comment (Xetiic) [inline README.md:58]: isnt this obsolete when using the yaml file for the environment? shoulr just state that the environment should be setup with the yaml file instead

File: README.md [hunk 2]
```diff
@@ -70,40 +70,44 @@ Images
   - AVIF [Pip Pillow AVIV Plugin](https://pypi.org/project/pillow-avif-plugin/)
 
     ```sh
-    pip install pillow-avif-plugin
+    pip install pillow-avif-plugin # version 1.3.1
     ```
 
   - HEIC [Pip Pillow HEIF Plugin](https://pypi.org/project/pillow-heif/)
```
Comment (Xetiic) [inline README.md:77]: see above

**Review by Xetiic — CHANGES_REQUESTED** [review]
Environment should be setup with the yaml file

**Review by m-langer — COMMENTED** [review]
removed separate installs and just state create Conda env

**Review by Xetiic — APPROVED** [review]

**Pull request merged**